dex/src/state.rs

```diff
@@ -1317,6 +1345,19 @@ pub enum EventView {
         fee_tier: FeeTier,
         client_order_id: Option<NonZeroU64>,
     },
+    MakerFill {
```

Comment:
Using this MakerFill doesn't actually do anything because the `EventView` is not what is stored in the `EventQueue` account. It's just a view of its data. So to add a field, one has to update the `Event` struct in a non breaking way.